plaidml/exec/BUILD

```diff
@@ -33,7 +33,9 @@ plaidml_cc_library(
     visibility = ["//visibility:public"],
     deps = [
         "//plaidml/core",
+        "//plaidml/edsl",
```

Comment:
Execution should not depend on edsl

Reply:
Hmm.  Something needs to know about both `Program` and `Executable`.  How would you like that constructed?

Reply:
Both of these are in `pmlc`, not part of the API surface.